packages/code-du-travail-frontend/src/Categories.js

```diff
@@ -3,7 +3,7 @@ import { Link } from "../routes";
 
 import { Container, Categories } from "@cdt/ui";
 
-import themes from "./data/themes.json";
+import themes from "../../code-du-travail-data/dataset/themes-front.json";
```

Comment:
:warning: This is kind of dangerous :warning: 
In a monorepo environment you should go through parent hierarchy *in the source code*!
The sources shouldn't be aware of the monorepo environment.
If you need to access to another package, use the created symlinks 

```diff
-import themes from "../../code-du-travail-data/dataset/themes-front.json";
+import themes from "code-du-travail-data/dataset/themes-front.json";
```